### PR TITLE
pakku 0.8.4 (new formula)

### PR DESCRIPTION
**Pull request by juraj-hrivnak**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/p/pakku.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  depends_on "openjdk"
+
+  def install
+    rm_f Dir["bin/*.bat"]
+    libexec.install %w[bin lib]
+    (bin/"Pakku").write_env_script libexec/"bin/Pakku", Language::Java.overridable_java_home_env
+  end
+
+  test do
+    assert_match "Could not read 'pakku-lock.json'", shell_output(bin/"Pakku add jei")
```
Comment (chenrui333) [inline Formula/p/pakku.rb:17]:
```suggestion
    assert_match "Could not read 'pakku-lock.json'", shell_output(bin/"Pakku add jei", 1)
```

File: Formula/p/pakku.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  url "https://github.com/juraj-hrivnak/Pakku/releases/download/v0.8.4/Pakku-0.8.4.tar"
+  sha256 "503e66145d3c525056b706b79804d6e741adcd6100714d6af3cd1d9caefc3a89"
+  license "EUPL-1.2"
+
+  depends_on "openjdk"
+
+  def install
+    rm_f Dir["bin/*.bat"]
+    libexec.install %w[bin lib]
+    (bin/"Pakku").write_env_script libexec/"bin/Pakku", Language::Java.overridable_java_home_env
```
Comment (chenrui333) [inline Formula/p/pakku.rb:13]:
```suggestion
    (bin/"pakku").write_env_script libexec/"bin/Pakku", Language::Java.overridable_java_home_env
```

**Comment by chenrui333** [comment]
@juraj-hrivnak thanks for the pr and can you squash the commits?

**Comment by juraj-hrivnak** [comment]
Sorry I messed up I will open new PR later

**Pull request closed without merging**